src/script/view_model/content/MessageListViewModel.ts

```diff
@@ -202,16 +204,12 @@ export class MessageListViewModel {
       return;
     }
 
-    const items: Message[] = await this.conversationRepository.getEventsForCategory(
-      this.conversation(),
-      MessageCategory.IMAGE,
-    );
-    const messageEntities = items.filter(
-      item => item.category & MessageCategory.IMAGE && !(item.category & MessageCategory.GIF),
-    );
-    const [imageMessageEntity] = messageEntities.filter(item => item.id === messageEntity.id);
-
-    amplify.publish(WebAppEvents.CONVERSATION.DETAIL_VIEW.SHOW, imageMessageEntity || messageEntity, messageEntities);
+    showDetailViewModal({
+      assetRepository: this.assetRepository,
+      conversationRepository: this.conversationRepository,
+      currentMessageEntity: messageEntity as ContentMessage,
```

Comment:
Is this as really needed or could it be resolved by stronger typing?

I really don't like the forced deactivation of the ts compiler. I stumbled on a lot of problems already caused by type assertions.